<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天津大学图书管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/TJU_logo.png</normaloff>:/images/TJU_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aaff;&quot;&gt;北洋大学图书馆&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;P&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;eiyang &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;L&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;ibrary &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;anagement &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;ystem&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_userinfo_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_bookSearch_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书检索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_returnBook_Button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>管理还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_userManagement_Button">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_manageBook_Button">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书管理</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="login_groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>系统登录</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="login_stuid_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>24</y>
       <width>128</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>24</y>
       <width>36</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>24</y>
       <width>128</width>
       <height>24</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>24</y>
       <width>36</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_action_button">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>72</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="main_user_status_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>34</y>
      <width>400</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>71</y>
      <width>316</width>
      <height>316</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/TJU_logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="main_logout_Button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>72</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <zorder>label_logo</zorder>
   <zorder>main_user_status_label</zorder>
   <zorder>label</zorder>
   <zorder>main_userinfo_Button</zorder>
   <zorder>main_bookSearch_Button</zorder>
   <zorder>main_returnBook_Button</zorder>
   <zorder>main_userManagement_Button</zorder>
   <zorder>main_manageBook_Button</zorder>
   <zorder>main_logout_Button</zorder>
   <zorder>login_groupBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="action_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
